bundles/framework/announcements/instance.js

```diff
@@ -0,0 +1,216 @@
+/**
+ * @class Oskari.framework.bundle.announcements.AnnouncementsBundleInstance
+ *
+ * Main component and starting point for the announcements functionality.
+ *
+ * See Oskari.framework.bundle.announcements.AnnouncementsBundleInstance for bundle definition.
+ */
+Oskari.clazz.define('Oskari.framework.bundle.announcements.AnnouncementsBundleInstance',
+
+    /**
+     * @method create called automatically on construction
+     * @static
+     */
+    function () {
+        this.sandbox = null;
+        this.started = false;
+        this.plugins = {};
+        this.localization = null;
+        this.seutumaisaSearchService = null;
```

Comment:
Looks like there's variables defined here that are not being used and/or refer to some specific service.